packages/ketrew/ketrew.3.1.0/opam

```diff
@@ -49,5 +49,6 @@ depopts: "postgresql"
 conflicts: [
   "ppx_inline_test" {>= "v0.9.0"}
   "ppx_sexp_conv" {>= "v0.9.0"}
+  "lwt" {>= "3.0.0" }
```

Comment:
I think this one is not needed (I already added `< "3.0.0"` in the depends fields a few days ago)